### DISCUSSION TITLE
Using Grafana for endpoints

### DISCUSSION THREAD

**Discussion by tarikus77** [Security-Onion-Solutions/securityonion#8115] [General] [answered]
Hi, I would like to use Security Onion as an endpoint resource monitoring as well (Using Grafana if possible). Is it possible?

**Comment by dougburks** [comment, marked as answer]
You might be able to make this work, but we don't provide any support for it. We only use Grafana for monitoring Security Onion itself. Most of our endpoint monitoring is focused on log collection:
https://docs.securityonion.net/en/2.3/host.html
